volatility/framework/layers/crash.py

```diff
@@ -42,7 +37,11 @@ def __init__(self, context: interfaces.context.ContextInterface, config_path: st
         self._context = context
         self._config_path = config_path
         self._page_size = 0x1000
-        self._base_layer = self.config["base_layer"]
+        try: 
+            self._base_layer = self.config["base_layer"]
+        except KeyError:
+            self._base_layer = 'base_layer'
```

Comment:
This doesn't seem like the right solution to this?  Better to set a default in the requirement, but then what would the default be?  I'm fine with logging the issue and providing a nicer error, but I don't think this will fix the problem?